src/js/view/controls/play-display-icon.js

```diff
@@ -42,34 +41,20 @@ export default class PlayDisplayIcon {
             } else {
                 iconDisplay.removeAttribute('aria-label');
             }
-
-            this.toggleIdleClass(newState, idleButtonText);
         });
 
-        this.toggleIdleClass(_model.get('state'), idleButtonText);
+        if (_model.get('displayPlaybackLabel')) {
+            let iconText = this.icon.getElementsByClassName('jw-idle-icon-text')[0];
+            if (!iconText) {
+                iconText = createElement(`<div class="jw-idle-icon-text"></div>`);
+                this.icon.appendChild(iconText);
+                addClass(this.icon, 'jw-idle-label');
+                iconText.textContent = localization.playback;
```

Comment:
You should always `appendChild` after making changes to the element, not before in the same function.